### DISCUSSION TITLE
Bridge problems

### DISCUSSION THREAD

**Discussion by vapsvapsov** [FuelLabs/sway#5288] [Q&A]
Hopefully that will be fixed soon, i honestly dont know the reason why fees are high wen you try to claim tokens and transactions do not succeed 

![image](https://github.com/FuelLabs/sway/assets/128076915/b67ec76d-794f-4cbe-ae02-58635c5e1ca8)

**Comment by Zeld5a** [comment]
Its OK now? I've had it hanging for more than a day
